.github/workflows/cs.yml

```
@@ -35,6 +35,9 @@ jobs:
           php-version: '7.4'
           coverage: none
           tools: cs2pr
+          github-token: ${{ secrets.PUBLIC_REPO_ACCESS }}
```

actionlint (1.7.7)

input "github-token" is not defined in action "shivammathur/setup-php@v2". available inputs are "coverage", "extensions", "ini-file", "ini-values", "php-version", "php-version-file", "tools"

(action)